src/content/en/updates/2016/10/navigator-share.md

```diff
@@ -2,8 +2,11 @@ project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
 description: Sharing is caring. Chrome is running an Origin Trial to enable native sharing on the web.
 
-{# wf_published_on: 2016-10-19 #}
-{# wf_updated_on: 2016-10-19 #}
+{# wf_published_on: 2016-10-20 #}
+{# wf_updated_on: 2016-10-20 #}
+{# wf_featured_image: /web/updates/images/generic/share.png #}
```

Comment:
@PaulKinlan - realized this should have had an icon, tags and a better description. I've added. will try to flag for next time.